### ISSUE TITLE
occur some errors when i use 2 GPUs

### ISSUE THREAD

**Issue by wemozj** [liujiaheng/compression#5]
When I train model using 2 GPUs in one linux machine, some errors occur. Maybe loss function have multiple dimensions.

![image](https://user-images.githubusercontent.com/40333940/83958940-31b3e000-a8aa-11ea-81da-71fdcc679417.png)
![image](https://user-images.githubusercontent.com/40333940/83958966-5ad47080-a8aa-11ea-9b62-ef4434885cb4.png)

**Comment by liujiaheng** [comment]
I don't run my code on multiple GPUs and I think this problem is easy to solve.
BTW, the experiments of ours are all based on 1 2080Ti GPU.
And the performance maybe drop when increasing the batch size.

**Comment by wemozj** [comment]
> I don't run my code on multiple GPUs and I think this problem is easy to solve.
> BTW, the experiments of ours are all based on 1 2080Ti GPU.
> And the performance maybe drop when increasing the batch size.

thank you

**Comment by LJ-lijian** [comment]
solve this problem?
